spec/go_runner_spec.vim

```diff
@@ -0,0 +1,38 @@
+source spec/support/helpers.vim
+
+describe "Go Runner"
+  before
+    cd spec/fixtures
+  end
+  
+  after
+    cd -
+  end
+
+  describe "when test#go#runner is not set"
+    after
+      call Teardown()
+    end
+    it "should use ginkgo if 'github.com/onsi/ginkgo' is found"
+      view ginkgo/normal_test.go
```

Comment:
Tests in this file do test against standard go test files. E.g.:
- [`view gotest/normal_test.go` on line 21](https://github.com/janko-m/vim-test/pull/186/files#diff-571e7dece54003e94224c6fda5045952R22) tests that gotest should be used if no ginkgo import is found
- [the for loop on line 30](https://github.com/janko-m/vim-test/pull/186/files#diff-571e7dece54003e94224c6fda5045952R30) tests that `go#runner` should be respected in both cases